R/zTest.R

```diff
@@ -214,17 +216,22 @@ safeZTest <- function(x, y=NULL, paired=FALSE, designObj=NULL,
     dataName <- paste(xLabel, "and", yLabel)
   }
 
+  if (is.null(ciValue))
+    ciValue <- 1-designObj[["alpha"]]
 
   result[["testType"]] <- testType
   result[["statistic"]] <- zStat
   result[["estimate"]] <- estimate
   result[["dataName"]] <- dataName
   result[["designObj"]] <- designObj
+  result[["ciValue"]] <- ciValue
 
-  result[["confSeq"]] <- computeZConfidenceSequence("nEff"=nEff, "meanStat"=meanStat,
-                                                    "phiS"=abs(designObj[["parameter"]]),
-                                                    "sigma"=sigma, "ciValue"=designObj[["ciValue"]],
-                                                    "alternative"="two.sided")
+  if (ciValue > 0 && ciValue < 1) {
```

Comment:
Throw error if this does not hold, earlier?

Reply:
Great idea